features.txt

```diff
@@ -649,6 +649,13 @@ tweaks.feather_falling_no_trample
 	desc:
 		Farmland will not get trampled when wearing feather falling.
 
+tweaks.no_sneak_bypass
+	name: No Sneak bypass
+	since: 1.3.1
```

Comment:
1.3.0 is the next release, not 1.3.1. Will fix this myself, not worth blocking the PR and making it your problem.